network-api/networkapi/templates/fragments/breadcrumbs.html

```diff
@@ -1,8 +1,8 @@
 
-<div id="breadcrumb-container">
+<div id="breadcrumb-container tw-pr-8">
```

Comment:
Adding this padding to the right of the container because I found on mobile screens the breadcrumbs and trailing "/" characters were getting very close to the "cutout".


**Before:**

![Screenshot 2023-06-19 at 15-42-37 Network effect hold view](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/92a3f81c-3a66-4aca-a6c0-a36b4b360836)


**After:**

![Screenshot 2023-06-19 at 15-42-25 Network effect hold view](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/e1b36b10-47fb-4fa4-8908-fb5e171fe7b9)